py3dtilers/Common/README.md

```diff
@@ -142,12 +142,18 @@ Project the features on another CRS. The `crs_in` flag allows to specify the inp
 | IfcTiler | :x: |
 | TilesetTiler | :heavy_check_mark: |
 
-Read the texture from the input and write it in the produced 3DTiles:
+Read the texture from the input and write it in image atlases (JPEG files):
 
 ```bash
 <tiler> <input> --with_texture
 ```
 
+You can choose the quality and the level of compression of the textures with the flags `--quality` and `--compress_level`. Quality varies between 1 and 100, where 100 is the maximum quality. Compression level varies between 0 and 9, where 9 is the maximum level and compression and where 0 doesn't compress the texture. By default, the quality of the textures is set to 95 and the compress level to 0.
```

Comment:
Just asking (open question, not mandatory to answer) : which parameter has the most impact on the size of the resulting image?  
Also, if we compress the image using the maximum level, how much time does it take to decompress? Is it worth to use the maximum level for rendering through the Web ?

Reply:
I still need to do some benchmarks, but the qualtiy seems to have a bigger impact on the size of the images. I also have to measure the impact of the compression on the 3DTiles loading time.

Reply:
Some benchmarks are available [here](https://github.com/VCityTeam/py3dtilers/issues/46#issuecomment-1140932365)